CHANGELOG.md

```diff
@@ -0,0 +1,125 @@
+### 1.8.0 - 2016-06-21
```

Comment:
Above the latest version, I’ve found it quite valuable to have an “Unreleased” section, which is where things that are on `master` are listed, but aren’t yet released in a published version. Two benefits:

1. People can quickly understand if they pull from `master`, how it differs from if they pull from a source like RubyGems
2. It makes it _much_ easier to put things in this Unreleased section as they are merged into `master`, then when a release is cut, just update the section title to the new version. You then don’t have to go through all the history and make a chore out of updating the change log when releasing.